ush/config_defaults.sh

```diff
@@ -1227,6 +1273,11 @@ NNODES_MAKE_ICS="4"
 NNODES_MAKE_LBCS="4"
 NNODES_RUN_FCST=""  # This is calculated in the workflow generation scripts, so no need to set here.
 NNODES_RUN_POST="2"
+NNODES_RUN_ANAL="16"
+#
+# Number of cores.
+#
+NCORES_RUN_ANAL="240"
```

Comment:
I think there should be some comments here on how to choose these. Are they specific to kjet? What if we need to use a different partition for a retro, or a different machine?

Reply:
On a second thought, It should be set as a small number as default. Set it to 4.